pandas/tests/scalar/test_timedelta.py

```diff
@@ -640,27 +640,6 @@ def conv(v):
         # invalid
         pytest.raises(ValueError, ct, '- 1days, 00')
 
-    def test_overflow(self):
```

Comment:
so rather than move these within the same file I would like to create another directory level (like we have for indexes) and put them in separate files, e.g.

```
pandas/tests/scalar/timedeltas/test_basic.py
.....test_slicing.py
```

etc. much more organized to have separate units of tests in separate named files (rather than classes). be sure to update setup.py with the additional directories.

Reply:
I'm amenable to that in general, but the relevantissue here is that the new classes (which I agree don't belong here) are not scalar tests at all.  Some are for DatetimeIndex/TimedeltaIndex which I can figure out destinations for.  I haven't looked at Series/DataFrame tests to see where those might go.  Or there could be something like tests/tslibs/vectorized/

Reply:
the would rather u do a PR which actually moves them to the right location rather than a temporary one

Reply:
Fair enough